docs/source/torch.compiler_faq.rst

```diff
@@ -543,33 +543,46 @@ Which NumPy features does ``torch.compile`` support?
 NumPy within ``torch.compile`` follows the latest NumPy release.
 
 Generally, ``torch.compile`` is able to trace through most NumPy constructions,
-and when it cannot, it fallbacks to eager and lets NumPy execute that piece of
+and when it cannot, it falls back to eager and lets NumPy execute that piece of
 code. Even then, there are a few features where ``torch.compile`` semantics
 slightly deviate from those of NumPy:
 
 - NumPy scalars: We model them as 0-D arrays. That is, ``np.float32(3)`` returns
-  a 0-D array under ``torch.compile``. For performance, it is best to use this 0-D
+  a 0-D array under ``torch.compile``. To avoid a graph break, it is best to use this 0-D
   array. If this is not possible, one may often recover the original behavior
-  by cast the NumPy scalar to the relevant Python scalar (e.g., ``float``).
+  by casting the NumPy scalar to the relevant Python scalar type ``bool/int/float``.
+
 - Negative strides: ``np.flip`` and slicing with a negative step return a copy.
+
 - Type promotion: NumPy's type promotion will change in NumPy 2.0. The new rules
   are described in `NEP 50 <https://numpy.org/neps/nep-0050-scalar-promotion.html)>`__.
   ``torch.compile`` implements NEP 50 rather than the current soon-to-be deprecated rules.
+
 - ``{tril,triu}_indices_from`` return arrays rather than lists of tuples.
 
-There are other feature for which we do not support tracing and we gracefully
+There are other features for which we do not support tracing and we gracefully
 fallback to NumPy for their execution:
 
 - Non-numeric dtypes like datetimes, strings, chars, void, structured dtypes and recarrays.
-- Long dtypes like ``np.float128`` and ``np.complex256``.
+
+- Long dtypes ``np.float128/np.complex256`` and some unsigned dtypes ``np.uint16/np.uint32/np.uint64`.
+
 - ``ndarray`` subclasses.
+
 - Masked arrays.
-- Ufunc machinery like ``axes=[(n,k),(k,m)->(n,m)]``, ``np.add.reduce``, etc.
-- Fortran orders and, in general, any ``order=`` different to ``C``.
+
+- Esoteric ufunc machinery like ``axes=[(n,k),(k,m)->(n,m)]`` and ufunc methods (e.g., ``np.add.reduce``).
+
+- Fortran ordered arrays and, in general, any ``order=`` different to ``C``.
```

Comment:
(sorry, did not catch it on the first read). 
Strictly speaking, some functions have defaults different from 'C' ('A' or 'K'). This does not matter in practice, and we graph break / raise NotImplementedError for non-default order arguments. 

Maybe just say any non-default values of the `order=` arguments?

Reply:
I'm just going to remove this line as we also graphbreak on a few other inputs and it'd be a pain to list them all.